src/ow1/heroes/orisa.opy

```diff
@@ -85,6 +86,7 @@ rule "[orisa.opy]: Trigger Supercharger start":
     @Condition eventPlayer.isHoldingButton(Button.ULTIMATE) == true
     @Condition eventPlayer.getUltCharge() == 100
     @Condition eventPlayer.isOnGround() == true
+    @Condition eventPlayer.is_using_supercharger == false
 
     startSupercharger()
```

Comment:
```suggestion

    endSupercharger() # Destroy any existing Supercharger
    startSupercharger() # Create new Supercharger
```

I think this more accurately represents the desired behavior. Your code makes it so that you can't bongo when using ult (which is a valid fix), but I know from playing 0 cd OW1 that placing a new bongo replaces the old bongo.

Looking at the code in `endSupercharger()`, I think this is safe to do because `destroyEffect(null)` safe.